Model/AutoRefreshBuff.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Threading;
+using System.Windows.Input;
+using System.Windows.Forms;
+using Newtonsoft.Json;
+using _4RTools.Utils;
+
+namespace _4RTools.Model
+{
+    public class AutoRefreshBuff : Action
```

Comment:
Autorefresh it not a buff, i guess `AutoKey` or only `AutoRefresh` is a better name.